python/outlines_core/fsm/outlines_core_rs.pyi

```diff
@@ -1,9 +1,8 @@
 from typing import Dict, List, Optional, Set, Tuple, Union
 
-def build_regex_from_schema(
-    json: str, whitespace_pattern: Optional[str] = None
-) -> str: ...
-def to_regex(json: Dict, whitespace_pattern: Optional[str] = None) -> str: ...
+def build_regex_from_schema(json: str, whitespace_pattern: Optional[str] = None) -> str:
```

Comment:
Nitpick: `json_schema` instead of `json`?

Reply:
This is the right time to update anything that bothers an eye, I will update 👍